src/algebra/ordered_monoid_lemmas.lean

```diff
@@ -361,6 +361,17 @@ calc  a * b ≤ a * 1 : mul_le_mul_left' h a
 
 end mul_one_class
 
+@[to_additive]
+lemma mul_left_cancel'' [semigroup α] [partial_order α]
+  [contravariant_class α α ((*) : α → α → α) (≤)] {a b c : α} (h : a * b = a * c) : b = c :=
```

Comment:
```suggestion
  [contravariant_class α α (*) (≤)] {a b c : α} (h : a * b = a * c) : b = c :=
```
I think that the type ascription is no longer needed: it was needed, at some point, when I experimented with making `α` implicit, but it seems that I forgot to remove all occurrences of this.